packages/google_identity_services_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.1+4
+
+* Downgrades SDK version to Dart `^3.3.0`.
```

Comment:
I would say "Restores support for Dart 3.3."; "downgrade" sounds like something is getting worse.

Reply:
Yep, better. Changed!